gm4_desire_lines/data/desire_lines/functions/init.mcfunction

```diff
@@ -0,0 +1,20 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing Desire Lines..."}]
+execute unless entity @p run say GM4: Installing Desire Lines...
+
+#declare and initialise scoreboards and settings
+scoreboard players set updateHappened gm4_upCheck 1
+scoreboard players set desire_lines gm4_modules 1
+scoreboard players set desire_lines gm4_clockTick 0
+
+scoreboard objectives add gm4_random dummy
```

Comment:
`gm4_random` should be reserved for future use if mojang ever adds a proper way to randomize integers. Even though the fake player is called `desire_lines`, the scoreboard name should still be unique, e.g. `gm4_dl_random`.

Reply:
Hmm I thought the `gm4_random` objective could be used for all kinds of PRNG's but I'll change it

Reply:
I see where you're coming from but I'm worried having just one scoreboard for different RNGs might cause issues in the future